daheinze-oscheer/mocking/countdown.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"time"
+)
+
+func main() {
+	sleeper := &ConfigurableSleeper{1 * time.Second, time.Sleep}
```

Comment:
I'm far away from this chapter. So maybe stupid question. What's the purpose of multiplication by 1?

Reply:
This is a very complicated arithmetic operation to confuse developers and hide the fact, that we are talking about one second here :-)